backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1921,6 +1921,28 @@ databaseChangeLog:
             columns:
               - column:
                   name: test_result_delivery
+  - changeSet:
+      id: add-no-phi-role
+      author: josh@skylight.digital
+      comment: Add a role for a user with reduced DB permissions (e.g., for Metabase)
+      changes:
+        - tagDatabase:
+          tag: add-no-phi-role
+        - sql:
+          sql: |
+            DO
+            '
+            BEGIN
+              IF NOT EXISTS (SELECT FROM pg_catalog.pg_roles WHERE rolname = ''${noPhiUsername}'') THEN CREATE ROLE ${noPhiUsername} WITH LOGIN NOSUPERUSER ENCRYPTED PASSWORD ''${noPhiPassword}'';
+              END IF;
+            EXCEPTION WHEN insufficient_privilege THEN
```

Comment:
I went to specifically check this since the migrations run fine, and it turns out...no, the migrations _don't_ run fine, it's just that `RAISE NOTICE` or `RAISE EXCEPTION` don't seem to actually signify a failure to Liquibase. Running the `CREATE ROLE` by itself lacks permission. I'm looking into whether there's a fix for this that isn't "just give the migration user the privileges" since then we won't catch this in non-local envs.

Reply:
I don't think there's a good way to do this without losing the ability to stop the migration on a real error, after looking at it. I think we'll have to add the migration manually to the `databasechangelog` tables of each DB after merge to essentially backport it. Commit changes made accordingly.

Reply:
Thanks! Can you make a ticket to encapsulate the manual migrations that will be required?